ai-ml/gke-a100-jax/README.md

```diff
@@ -1,7 +1,3 @@
 # JAX 'Hello World' on GKE + A100-80GB
 
-[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/kubernetes-engine-samples&cloudshell_tutorial=README.md&cloudshell_workspace=ai-ml/gke-a100-jax)
-
-This tutorial shows how to run a simple JAX program using NVIDIA GPUs A100-80GB on a GKE cluster
-
-Visit https://cloud.google.com/blog/products/containers-kubernetes/machine-learning-with-jax-on-kubernetes-with-nvidia-gpus to follow the tutorial
+This tutorial has been moved to the [ai-on-gke](https://github.com/GoogleCloudPlatform/ai-on-gke) repo.
```

Comment:
Has any tutorial / docs pointing to this repo been already updated to point to the new repo?